examples/2node-ixia-ceos.pb.txt

```diff
@@ -0,0 +1,50 @@
+name: "ixia-c"
+nodes: {
+    name: "otg"
+    type: IXIA_TG
+    version: "0.0.1-2770"
```

Comment:
you should use the magic version of 0.0.1-9999 
as this is used internally to mean "latest"

Reply:
https://github.com/google/kne/pull/138/commits/f322e580c4df8273520c0f98be4dc6397b568ef0